### PR TITLE
better responses 💬

### PR DESCRIPTION
**Pull request by KartikeSingh**
- fixed the issue
- added more responses

### CODE REVIEW

File: minichat/core.py [hunk 1]
```diff
@@ -15,7 +15,29 @@ def __init__(self) -> None:
             "I don't know D:",
             "D:",
             "What?",
-            "*thinks*"
+            "*thinks*",
+            "Interesting, Indeed!",
```
Comment (null8626) [inline minichat/core.py]: change this to `Interesting indeed!`

File: minichat/core.py [hunk 2]
```diff
@@ -15,7 +15,29 @@ def __init__(self) -> None:
...
+            "Are you okay ? (mentally)",
+            "If its now worth it, Don't do it",
+            "-_- bruh, are you stupid",
+            "no that is not good. YOU BETTER DON'T",
+            "baka, are you serious ?",
+            "wow that is pretty cool",
+            "Maybe you should follow your brain",
+            "i think ask your father, kiddo",
+            "You can't do this leave it",
+            "Kiddo go and complete your maths homeowork first"
```
Comment (null8626) [inline minichat/core.py]: try `go complete` instead of `go and complete`

File: minichat/core.py [hunk 3]
```diff
@@ -15,7 +15,29 @@ def __init__(self) -> None:
...
+            "No your PP is too small for this",
+            "ig, You don't have a life",
```
Comment (null8626) [inline minichat/core.py]: the comma here is not needed

File: minichat/core.py [hunk 4]
```diff
@@ -15,7 +15,29 @@ def __init__(self) -> None:
...
+            "I will research about that",
+            "tell me a joke instead xD",
+            "Wait, I am trying to make sense",
+            "Dude you are disgusting",
+            "Do you have sense? oh yea you don't xD",
```
Comment (null8626) [inline minichat/core.py]: try `you don't make any sense` for better wording.

**Review by null8626 — CHANGES_REQUESTED** [review]

**Comment by KartikeSingh** [comment]
okay null chan

**Comment by null8626** [comment]
thanks! <3

**Review by twlite — CHANGES_REQUESTED** [review]
We are not tryna roast users here, please update the responses.

**Comment by KartikeSingh** [comment]
> We are not tryna roast users here, please update the responses.

okay xD

**Comment by KartikeSingh** [comment]
my messages are too cringe, i feel like closing the pr, sorry for weird commits 💀

**Pull request closed without merging**